src/components/RadioCard/RadioCard.types.ts

```diff
@@ -4,16 +4,19 @@ export type InputEvent = Event
 export type RadioCardProps = {
   checked: boolean
   className?: string
+  content: string | Component
+  heading?: string
```

Comment:
Since `getHeadingMarkup` also handles receiving a function, should this be `string | Component`?

Reply:
good catch, I did improve the typing in 9f771542